src/tokenOperations/tokenOperationsFactory.ts

```diff
@@ -25,6 +26,8 @@ interface IConfig {
     gasLimitStorePerByte: IGasLimit;
     issueCost: BigNumber.Value;
     esdtContractAddress: IAddress;
+
+    shouldWarnAboutUnsettingBurnRoleGlobally: boolean;
```

Comment:
In theory, this is a breaking change. However, in practice, it should not be. Generally speaking, users would be using the default config implementation, `TokenOperationsFactoryConfig`. Furthermore, this transactions factory isn't widely used yet.